File: 0x08-python-more_classes/3-rectangle.py
#!/usr/bin/python3

class Rectangle:
    rect = Rectangle(5, 10)
print(rect.area())  # Output: 50
print(rect.perimeter())  # Output: 30

rect.width = 7
rect.height = 12

print(rect.area())  # Output: 84
print(rect.perimeter())  # Output: 38

rect.width = 0
rect.height = 5

print(rect.area())  # Output: 0
print(rect.perimeter())  # Output: 10

File: 0x13-javascript_objects_scopes_closures/3-rectangle.js
#!/usr/bin/node

const rectangle1 = new Rectangle(5, 4);
rectangle1.print();
// Output:
// XXXXX
// XXXXX
// XXXXX
// XXXXX

const rectangle2 = new Rectangle(0, 5);
rectangle2.print();
// Output:
// Empty object

const rectangle3 = new Rectangle(3.5, 2);
rectangle3.print();
// Output:
// Empty object
